src/sentry/static/sentry/app/components/forms/multiSelectControl.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import SelectControl from 'app/components/forms/selectControl';
+
+export default class MultiSelectControl extends React.Component {
+  render() {
+    return (
+      <SelectControl
+        style={{width: 200, overflow: 'visible'}}
```

Comment:
I don't think this is needed here, but it was leftover from `<MultiSelectField>`